dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/ExportServiceTest.java

```diff
@@ -0,0 +1,485 @@
+        depositorFieldType.setTitle("Depositor");
+        depositorFieldType.setDisplayOrder(6);
+
+        DatasetFieldType dateOfDepositFieldType = MocksFactory.makeDatasetFieldType("dateOfDeposit", FieldType.TEXT, false, citationMetadataBlock);
+        dateOfDepositFieldType.setTitle("Deposit Date");
+        dateOfDepositFieldType.setDisplayOrder(7);
+        dateOfDepositFieldType.setUri("http://purl.org/dc/terms/dateSubmitted");
+
+        when(datasetFieldService.findByNameOpt(eq("title"))).thenReturn(titleFieldType);
+        when(datasetFieldService.findByNameOpt(eq("author"))).thenReturn(authorFieldType);
+        when(datasetFieldService.findByNameOpt(eq("authorName"))).thenReturn(authorNameFieldType);
+        when(datasetFieldService.findByNameOpt(eq("authorAffiliation"))).thenReturn(authorAffiliationFieldType);
+        when(datasetFieldService.findByNameOpt(eq("datasetContact"))).thenReturn(datasetContactFieldType);
+        when(datasetFieldService.findByNameOpt(eq("datasetContactName"))).thenReturn(datasetContactNameFieldType);
+        when(datasetFieldService.findByNameOpt(eq("datasetContactAffiliation"))).thenReturn(datasetContactAffiliationFieldType);
+        when(datasetFieldService.findByNameOpt(eq("datasetContactEmail"))).thenReturn(datasetContactEmailFieldType);
+        when(datasetFieldService.findByNameOpt(eq("dsDescription"))).thenReturn(dsDescriptionFieldType);
+        when(datasetFieldService.findByNameOpt(eq("dsDescriptionValue"))).thenReturn(dsDescriptionValueFieldType);
+        when(datasetFieldService.findByNameOpt(eq("dsDescriptionDate"))).thenReturn(dsDescriptionDateFieldType);
+        when(datasetFieldService.findByNameOpt(eq("subject"))).thenReturn(subjectFieldType);
+        when(datasetFieldService.findControlledVocabularyValueByDatasetFieldTypeAndStrValue(subjectFieldType, "Chemistry", false))
+                .thenReturn(subjectFieldType.getControlledVocabularyValue("chemistry"));
+        when(datasetFieldService.findControlledVocabularyValueByDatasetFieldTypeAndStrValue(subjectFieldType, "Agricultural Sciences", false))
+                .thenReturn(subjectFieldType.getControlledVocabularyValue("agricultural_sciences"));
+        when(datasetFieldService.findControlledVocabularyValueByDatasetFieldTypeAndStrValue(subjectFieldType, "Arts and Humanities", false))
+                .thenReturn(subjectFieldType.getControlledVocabularyValue("arts_and_humanities"));
+
+        when(datasetFieldService.findByNameOpt(eq("depositor"))).thenReturn(depositorFieldType);
+        when(datasetFieldService.findByNameOpt(eq("dateOfDeposit"))).thenReturn(dateOfDepositFieldType);
+    }
+
+    private String readFileToString(String resourcePath) throws IOException, URISyntaxException {
+        byte[] bytes = Files.readAllBytes(Paths.get(getClass().getClassLoader().getResource(resourcePath).toURI()));
+        return new String(bytes);
```

Comment:
You can achieve the same effect with `IOUtils.resourceToString(classpath, charset)`